### DISCUSSION TITLE
Combining `members: true` with filtering

### DISCUSSION THREAD

**Discussion by mhuebert** [mkdocstrings/python#201] [Q&A]
I'd like to show documentation for all members of a module (including those imported from other modules), while filtering out members that don't have docstrings & those beginning with `_`. 

Setting `members: true` is the only way I've found to include imported members, yet it has the effect of ignoring filters & docstring settings. 

Eg. the following will show everything:
```
options:
  members: true
  show_if_no_docstring: false # has no effect
```

Is there another way, other than setting `members: true`, to include imported members?

**Comment by pawamoy** [comment]
There's no way to to select/exclude imported objects yet, but we track this here: https://github.com/mkdocstrings/python/issues/170 :slightly_smiling_face: 

Note however that if you explicitly mark these imported objects as public, they'll be rendered. You can add their names to the module's `__all__` list.
